signer/src/storage/postgres.rs

```diff
@@ -1749,6 +1750,60 @@ impl super::DbRead for PgStore {
 
         self.get_sweep_transaction_by_txid(&txid).await
     }
+
+    async fn get_sweep_transaction_package(
+        &self,
+        prevout_txid: &model::BitcoinTxId,
+    ) -> Result<Vec<model::SweepTransaction>, Error> {
+        let infos: Vec<(model::BitcoinTxId, i32)> = sqlx::query_as(
+            "
+            WITH RECURSIVE sweep_txs AS (
+                SELECT
+                    txid
+                  , signer_prevout_txid
+                  , 1 AS number
+                FROM 
+                    sweep_transactions
+                WHERE 
+                    txid = $1
+
+                UNION ALL
+
+                SELECT
+                    tx.txid
+                  , tx.signer_prevout_txid
+                  ,  last.number + 1
+                FROM
+                    sweep_transactions tx
+                INNER JOIN
+                    sweep_txs last 
+                        ON tx.signer_prevout_txid = last.txid
+            )
+            SELECT
+                txid
+              , number
+            FROM
+                sweep_txs
+            ORDER BY
+                number ASC;
```

Comment:
Doesn't this find all transactions after the `prevout_txid`? I mean, including the ones issued as part of subsequent transaction packages?

Reply:
Also, this returns the transaction that includes the given txid. This isn't bad per se, but when computing the last fee we'll include a transaction that has been confirmed, which will give us a bad last fee. And if we have RBF'ed a transaction package more than once, this will include all packages not just the one that we care about, which is the last one.

Reply:
Yeah you're right, there's some funk here. I'll add a couple negative tests to ensure those scenarios can't happen and update the query.

Reply:
I updated the query which I think solves the above issues (and current tests pass), but I still need to write the negative tests